src/olympia/reviewers/utils.py

```diff
@@ -375,7 +375,9 @@ def get_actions(self, request):
             request.user, permission)
 
         # Special logic for availability of reject multiple action:
-        if self.content_review:
+        if (self.content_review or
+                is_recommendable or
+                self.addon.type == amo.ADDON_STATICTHEME):
```

Comment:
I can see how the change fixes the linked issues but trying to read this function from beginning to end to work out what actions would be available makes my head hurt.  

Suggestion for follow-up: each action should be split into a class and we can move the logic around if its available into there; this function can be something like  `return [action.name: action for action in [ActionClass(...) for ActionClass in self.actions]]`

Reply:
I'm not sure that would be a good thing. The availability conditions for the various actions share a lot of code, and that's why they have been kept in the same method. When I refactored the permission checks I started by separating the checks, and it ended up making things a lot more complicated because I still wanted to avoid copy/pasting a bunch of code around.